source/deploy/consider-mobile-vpn-options.rst

```diff
@@ -45,7 +45,7 @@ Connect via Corporate Proxy Server
 
 Review the following commonly-asked questions about connecting through a corporate proxy server:
 
-- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
-- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://docs.mattermost.com/mobile/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment>`__
-- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://docs.mattermost.com/mobile/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server>`__
-- `Run App Store versions of the Mattermost Mobile apps <https://docs.mattermost.com/mobile/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps>`__
+- `How do I receive mobile push notifications if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/deploy/mobile-faq.html#how-do-i-receive-mobile-push-notifications-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`__
+- `Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment <https://developers.mattermost.com/contribute/mobile/push-notifications/corporate-proxy/#receiving-mobile-push-notification-if-the-use-of-a-corporate-proxy-server-is-required-by-your-it-policy>`__
+- `Whitelist Mattermost push notification proxy to bypass your corporate proxy server <https://developers.mattermost.com/contribute/mobile/push-notifications/corporate-proxy/#receiving-mobile-push-notification-if-the-use-of-a-corporate-proxy-server-is-required-by-your-it-policy>`__
+- `Run App Store versions of the Mattermost Mobile apps <https://developers.mattermost.com/contribute/mobile/push-notifications/corporate-proxy/#receiving-mobile-push-notification-if-the-use-of-a-corporate-proxy-server-is-required-by-your-it-policy>`__
```

Comment:
This link is repeated - is it relevant to both points?

Reply:
@cwarnermm, these four links didn't have 301s and I'm not sure where they're meant to point (but the dev docs don't seem to be a perfect fit).

Reply:
Thanks @Sayanta66 and @justinegeffen!

This Docathon 2021 contribution has revealed that previously-existing content is missing from the current doc set. I've re-added this content via https://github.com/mattermost/docs/pull/4871

Once this new PR is merged, the 5 links above in @Sayanta66's PR should be updated as follows:
- deploy/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server
- deploy/mobile-faq.html#deploy-mattermost-in-a-proxy-aware-configuration-with-a-pre-proxy-relay
- deploy/mobile-faq.html#deploy-mattermost-with-connection-restricted-post-proxy-relay-in-dmz-or-a-trusted-cloud-environment
- deploy/mobile-faq.html#whitelist-mattermost-push-notification-proxy-to-bypass-your-corporate-proxy-server
deploy/mobile-faq.html#run-app-store-versions-of-the-mattermost-mobile-apps